server/conf/conf.go

```diff
@@ -157,6 +163,12 @@ type NATSStreamingConfig struct {
 	ConnectWait        int // milliseconds
 }
 
+// JetStreamConfig configuration for a JetStream connection
+type JetStreamConfig struct {
+	PublishAsyncMaxPending int
```

Comment:
I think this one maybe optional, since could change the order of the stream?

Reply:
If not explicitly configured, then this won't be used. I added this in JetStream because it seemed similar to `MaxPubAcksInflight` for STAN.